addons/website_slides_survey/models/res_partner.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    certifications_count = fields.Integer('Succeeded Certifications Count', compute='_compute_certifications_count')
+    company_certifications_count = fields.Integer('Succeeded Certification By Company Partners', compute='_compute_company_certifications_count')
```

Comment:
same here, name it : certifications_company_count

Reply:
+1 see hereabove  same comments apply :)